test/Msbuild.Tests.Utilities/Msbuild.Tests.Utilities.csproj

```diff
@@ -2,10 +2,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp1.6</TargetFramework>
```

Comment:
ok i think i got confused here, i thought that you can only reference netstandard1.6 lib from netstandard1.6 but the numbers do not seem to be correlated in any way. ill revert this. i changed this because i started getting some really not clear errors